Doc/runtime.txt

```diff
@@ -0,0 +1 @@
+3.7
```

Comment:
Could this be specified in netlify.toml too instead of a special-purpose file?

Reply:
Nope. https://www.netlify.com/docs/build-settings